src/completion-listener.ts

```diff
@@ -43,15 +43,18 @@ export class CompletionListener {
             return events[0].exitCode === 0;
         } else if (this.successCondition === 'last') {
             return events[events.length - 1].exitCode === 0;
-        } else if (!/^!?command-.+$/.test(this.successCondition)) {
+        }
+
+        const commandSyntaxMatch = this.successCondition.match(/^!?command-(.+)$/);
+        if (commandSyntaxMatch === null) {
```

Comment:
nitpick, if you like the suggestion...

```suggestion
        if (commandSyntaxMatch == null) {
```